compiler/testData/diagnostics/tests/controlStructures/forWithNullableIterator.fir.kt

```diff
@@ -8,5 +8,5 @@ class It {
 }
 
 fun test() {
-  <!INAPPLICABLE_CANDIDATE, INAPPLICABLE_CANDIDATE!>for (x in Coll()) {}<!>
+  <!UNSAFE_CALL, UNSAFE_CALL!>for (x in Coll()) {}<!>
```

Comment:
For my own record, these should be rather `HAS_NEXT_FUNCTION_NONE_APPLICABLE` and `NEXT_NONE_APPLICABLE`: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/controlStructures/forWithNullableIterator.kt#L11